app/services/stat_retrieval_service.rb

```diff
@@ -22,6 +22,7 @@ def stats_string(user)
     def rankings(limit: 1000)
       ranked_users = []
       User.where('rank != 1500').order('rank desc').limit(limit).each do |u|
```

Comment:
This doesn't pertain to your introduction, but it looks like anyone who is ranked 1500 isn't counted. Isn't it valid that someone is ranked and it happens to be 1500?